release_notes.md

```diff
@@ -1,22 +1,19 @@
 # What's New
 
-## Playbook Enhancements
+1. **Disabling `Simulate Scenario` Button During Execution:**
+   - We've added a feature where the "Simulate Scenario" button is disabled while executing the scenario playbook. This prevents accidental triggering of simulations during active scenario execution.
```

Comment:
We can remove this point from the release notes as this issue is partially fixed